include/IrrCompileConfig.h

```diff
@@ -47,7 +47,7 @@
 //! Example: NO_IRR_COMPILE_WITH_X11_ would disable X11
 
 //! Uncomment this line to compile with the SDL device
-//#define _IRR_COMPILE_WITH_SDL_DEVICE_
+#define _IRR_COMPILE_WITH_SDL_DEVICE_

```

Comment:
If you want you can add a CI run that compiles and tests this, but it'll have to stay disabled by default.

Reply:
I forgot this was enabled. I meant to keep it commented out.